src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -14,6 +15,30 @@
 
 namespace Dynamo.Configuration
 {
+    static class ExtensionMethods
+    {
+        /// <summary>
+        /// Copy Properties from a PreferenceSettings instance to another
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="destination"></param>
+        public static void CopyProperties(this PreferenceSettings source, PreferenceSettings destination)
```

Comment:
can we make this internal?

Reply:
looks like the class is internal (by default)
But it making the method itself internal makes it clearer I guess